spec/support/factory_girl.rb

```diff
@@ -21,4 +21,29 @@
     password 'password'
     password_confirmation 'password'
   end
+
+  factory :rental_unit do
+    address { "#{Faker::Address.street_address}, #{Faker::Address.city},
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.